patches/chrome-browser-importer-external_process_importer_host.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/browser/importer/external_process_importer_host.cc b/chrome/browser/importer/external_process_importer_host.cc
+index 9087093838f249ca73cc9fc82d54d5b44e9bb5f2..22e47149317aa6f7a48661d58156f176a2be33e8 100644
+--- a/chrome/browser/importer/external_process_importer_host.cc
++++ b/chrome/browser/importer/external_process_importer_host.cc
+@@ -60,6 +60,7 @@ void ExternalProcessImporterHost::StartImportSettings(
+     return;
+   }
+ 
++  BRAVE_START_IMPORT_SETTINGS
+   CheckForLoadedModels(items);
+ 
+   LaunchImportIfReady();
+@@ -113,6 +114,7 @@ void ExternalProcessImporterHost::LaunchImportIfReady() {
+   client_ = new ExternalProcessImporterClient(
+       weak_ptr_factory_.GetWeakPtr(), source_profile_, items_, bridge);
+   client_->Start();
++  BRAVE_LAUNCH_IMPORT_IF_READY
```

Comment:
same as above and I definitely think we should not be patching for p3a, we can find another place to do that

Reply:
How about leaving this as a f/u? This goes  beyond of the scope of this PR.
This PR just does cleanup current patching around importer area.

Reply:
fixed by https://github.com/brave/brave-core/pull/4672/commits/8c029feb736f5134caa3f6a3f398ab404b7d4cfd